core/src/avm1/activation.rs

```diff
@@ -237,6 +237,8 @@ pub struct Activation<'a, 'gc: 'a, 'gc_context: 'a> {
     /// This is often the name of a function (if known), or some static name to indicate where
     /// in the code it is (for example, a with{} block).
     pub id: ActivationIdentifier<'a>,
+
+    times_get_time_called: u32,
```

Comment:
I believe this should be in context as right now it'll fail if there's a function call between the busy loop and the actual time call

Reply:
I'm still afraid this might increase false positives, but done :+1: